### PR TITLE
Update README.md with note about main branch

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?

Add one of the following kinds:
* documentation
* subproject management

#### What this PR does / why we need it:

Update the README.md with the note about the main branch (frequent changes) and the hint to rely only on published releases. As requested in last QoD community call.

#### Which issue(s) this PR fixes:

n/a

Fixes #

#### Special notes for reviewers:

Sentence copied over from the CHANGELOG.md, just adapted the highlighting.

### CODE REVIEW

**Comment by hdamker** [comment]
As discussed within last QoD call. @tlohmar please have a look if it is fulfilling your expectations.

**Review by shilpa-padgaonkar — APPROVED** [review]

File: README.md
Comment (tlohmar) [inline README.md:1]: Since main and all branches are publicly available, it might be better to write about "stable releases" (not "published releases"

Comment (tlohmar) [reply README.md:1]: Hmm, the source of the confusion seems to be the sentence "The latest published release and version of Quality-On-Demand (QoD) API is 0.8.1.". Version 0.8.1 is only available in a branch and 0.8.1 is actually not a release.

File: README.md [hunk 1]
```diff
@@ -23,9 +23,10 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
...
 * Meeting link: <a href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_MGM3YTBjYWYtNGVmNy00Mjk1LWJhMTktZjI1M2NjNzg2ZDFh%40thread.v2/0?context=%7b%22Tid%22%3a%22bde4dffc-4b60-4cf6-8b04-a5eeb25f5c4f%22%2c%22Oid%22%3a%2237ff36be-0e4d-42c3-ac06-7b904f0f6b24%22%7d">Microsoft Teams Meeting</a> 
 
-## Results and Status
-* The latest release and version of Quality-On-Demand (QoD) API is 0.8.1. This is the seconed alpha version of the API. There are bug fixes to be expected and incompatible changes in upcoming versions. It is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
-* Version 0.8.1 of the API :
+## Status and released versions
+* Note: Please be aware that the project will have frequent updates to the **main** branch. **There are no compatibility guarantees associated with code in any branch, including main, until it has been released.** For example, changes may be reverted before a release is published. **For best results, use the latest published release**.
```
Comment (tlohmar) [inline README.md:27]: I think, we should re-phrase "...until it has been released" to ""...until a new intermedia release is created". 
"For example, changes may be reverted before a release is **published**." --> For example, changes may be reverted before a release is **created**.
"**For best results, use the latest published release**." --> **For best results, use the latest available release**.

File: README.md [hunk 2]
```diff
@@ -23,9 +23,10 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
...
+* **The latest published release and version of Quality-On-Demand (QoD) API is 0.8.1. This is the seconed alpha version of the API.** There are bug fixes to be expected and incompatible changes in upcoming versions. It is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
```
Comment (tlohmar) [inline README.md:28]:
```suggestion
* **The latest available release and version of Quality-On-Demand (QoD) API is 0.8.1. This is the second alpha version of the API.** There are bug fixes to be expected and incompatible changes in upcoming releases. It is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
```

**Comment by hdamker** [comment]
@tlohmar thanks - got the issue and committed your suggestions.

**Review by shilpa-padgaonkar — DISMISSED** [review]

**Comment by hdamker** [comment]
Minor edit for readability ... ready to merge? @tlohmar @shilpa-padgaonkar

**Comment by tlohmar** [comment]
> Minor edit for readability ... ready to merge? @tlohmar @shilpa-padgaonkar

Looks good.

**Pull request merged**